### ISSUE TITLE
Clear formatting not working on Windows

### ISSUE THREAD

**Issue by smaudet** [zim-desktop-wiki/zim-desktop-wiki#2401]
**Describe the bug**
Clear formatting not working on Windows. There are two parts:
Keyboard shortcut does not work.
When it does work (via menu), it still does not clear link formatting.

**To Reproduce first issue**
1. Format some text
2. Select the text, or put the cursor in the middle of the formatted text
3. Use keyboard shortcut Ctrl+9
4. Text formatting does not clear

**To Reproduce second issue**
1. Write SomethingSomething and put a space.
2. Navigate back to the entered text
3. Attempt to clear the formatted link by keyboard shortcut
4. Attempt to clear the formatted link by menu option.
6. Link formatting does not clear or remove.

**Expected behavior**
Clear formatting should work via keyboard shortcuts. Other keyboard shortcuts, e.g. Ctrl+1, work fine. It is possible this is partly on Windows, this behavior is seen on earlier versions of Zim which used to work fine.

Clearing a formatted link should also remove the empty page. This is the prior behavior, empty junk pages should not be left around simply because some CamelCase text was left around.

**Screenshots**
![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/990607/b321a2b3-336c-4498-937c-8b388ac8f4db)

**Environment (please complete the following information):**
 - Version 0.75.1
 - OS: Windows 10 19041
 - Language: EN/US

**Additional context**

**Comment by jaap-karssenberg** [comment]
Just tested this and clearing formatting (bold, italic, ...) by either key
binding or menu item works fine.

What you are trying to do is remove a link. In zim, links are not treated
as text formatting but have their own "remove link" item in the context
menu.

-- Jaap

**Comment by smaudet** [comment]
> What you are trying to do is remove a link

@jaap-karssenberg that would cover half of the bug, however for me also the Ctrl+9 shortcut just does not work. Perhaps you could mention which logs to collect, or maybe I need to build this for Windows to see if it even gets the shortcut key or not (other Ctrl+<number> keys work, just Ctrl+9 does not).

> In zim, links are not treated
as text formatting but have their own "remove link" item in the context
menu.

Fair enough, but it *did* use to work this way. I suppose maybe this feature was intentionally removed, and I should convert this half of the "bug" into a "feature request".

Regardless, this is UX/keybinding issue - there is no key-bound to the remove link (which does work from the context menu), so there is no way to say "hey zim I just want this to be plain text, no really". I understand separate methods to clear was probably cleaner from a code perspective, but not a usability one.

**Comment by introt** [comment]
>  which logs to collect

Tools > View debug log

should contain info on whether the key binding triggers the function

**Comment by mbluett88** [comment]
> Just tested this and clearing formatting (bold, italic, ...) by either key binding or menu item works fine. What you are trying to do is remove a link. In zim, links are not treated as text formatting but have their own "remove link" item in the context menu.
> […](#)

In a previous version (0.72.0), Ctrl-9 worked just fine. So, we have become accustom to this working. Now it no longer works. This seems to be the problem. It was working and now no longer does. It is a lot faster for me to use the keyboard than having to switch to the mouse all the time. So, a Ctrl key option would be much nicer. 

 Other than this one thing I think the way Zim works is great. 

It has provided an excellent method for note organization. 

I also use it to organize my electronic library and make notes on books as I read them. So, it is a very helpful app. I am slowly converting my massive collection of separate note files into Zim so that I have one main source I can search on.

UPDATE:  Actually the Ctrl-9 function can be re-enabled:  Edit > Preferences > Key Bindings then scroll down to **Actions/Notebook View/clear_formatting**   and change this to use Ctrl-9.

**Comment by 1ras** [comment]
I agree, it's IMHO a step backward, e.g. it took me about 30 minutes to find this bug report and another 10 minutes just to find the "Remove Link" option in the menu, because it is below Edit where all other formatting options like "Remove Heading", "Remove List" and "Clear Formatting" are below Format...

In the past it was very easy to get every formatting removed, just mark the text and press Ctrl+9. This also cleared the heading and the link.

I am a bit surprised of the use case for replacing one handy clear formatting option with multiple options. I mean if I don't want to clear a link I just don't mark it before pressing Ctrl+9. Same if I don't want to clear the heading. Having to remember now three or even more keyboard shortcuts (and Remove Link has not even a keyboard shortcut) sounds IMHO not very user friendly.

Just to note, this bug also affects Linux (subject says Windows).
